prometheus-exporters/vrops-exporter/alerts/host.alerts

```diff
@@ -6,7 +6,7 @@ groups:
       vrops_hostsystem_runtime_connectionstate{state="notResponding"}
       and on (hostsystem) vrops_hostsystem_runtime_powerstate{state="Unknown"}
       and on (hostsystem) vrops_hostsystem_runtime_maintenancestate{state="notInMaintenance"}
-      and on (hostsystem) vrops_hostsystem_summary_running_vms_number > 0
+      and on (hostsystem) count(vrops_virtualmachine_runtime_powerstate{state="Powered On"}) by (hostsystem) > 0
```

Comment:
Do we really need a grouping here? The comparison is on hostsystem level anyways.

Reply:
No, we don't need it. But is is also not wrong.

Reply:
You could suggest a change if you like.